config/base.json

```diff
@@ -43,6 +43,7 @@
 		"enable_auth": false,
 		"enable_datadog": true,
 		"enable_io_beta_cta": true,
-		"enable_product_docs_search": true
+		"enable_product_docs_search": true,
+		"enable_global_search": false
```

Comment:
💭 you could set this as true and then only set production as 'false'. Same same, just thinking outloud.